tests/db/compact_db_migrations.py

```diff
@@ -0,0 +1,99 @@
+"""
+    Copyright 2023 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+
+
+    Tool to compact db migration scripts
+"""
+import asyncio
+import importlib
+import os
+import re
+
+import asyncpg
+import pytest
+
+UP_TO_VERSION = "202206290"  # Specify the version up to which migrations should be applied
+
+MIGRATIONS_DIR = "../../src/inmanta/db/versions"
```

Comment:
Like this the script will fail/succeed depending on the current working directory you use to execute the script.

```suggestion
MIGRATIONS_DIR = f"{os.path.dirname(os.path.dirname(__file__))}/src/inmanta/db/versions"
```